zpaq: add page (#15024)
tldr-pages/tldr: pages/common/zpaq.md

# zpaq

> > Incremental Journaling Backup Utility and Archiver.

Reviewer: ```suggestion
> Incremental journaling backup utility and archiver.
```

> More information: <https://mattmahoney.net/dc/zpaq.html>.

- [a]dd a file or directory to a new or existing archive:

`zpaq a {{path/to/archive.zpaq}} {{path/to/file_or_directory}}`

> - Create or add to encrypted archive:

Reviewer: ```suggestion
- Create or add to an encrypted archive:
```

> - Create or add to encrypted archive with password X:
> 
> `zpaq a -k{{X}} {{path/to/archive.zpaq}} {{path/to/file_or_directory}}`

Reviewer 2: ```suggestion
- Create or add to encrypted archive:

`zpaq a -k{{password}} {{path/to/archive.zpaq}} {{path/to/file_or_directory}}`
```

> - E[x]tract the most recent recent versions of files:

Reviewer: ```suggestion
- E[x]tract the most recent versions of files:
```

Author: Wow I'm amazed I missed that

`zpaq x {{path/to/archive.zpaq}}`

> - [l]ist or compare external files to archive by dates:

Reviewer: ```suggestion
- [l]ist the archive contents:
```

I think this makes it clearer for non-advanced users.

`zpaq l {{path/to/archive.zpaq}}`

- Set the level of compression (higher means more compression but slower):

`zpaq a {{path/to/archive.zpaq}} -m{{1|2|3|4|5}} {{path/to/file_or_directory}}`

> - E[x]tract to directory as of the date or version number:
> 
> `zpaq x {{path/to/archive.zpaq}} {{path/to/extract}} -to {{path/to/output}} -until {{YYYY-MM-DD}}`

Reviewer: ```suggestion
- E[x]tract the specified files from the archive that are not newer than the specified date:

`zpaq x {{path/to/archive.zpaq}} {{path/in/archive/to/extract}} -to {{path/to/output}} -until {{YYYY-MM-DD}}`
```
